docs/CONTRIBUTING.md

```diff
@@ -94,6 +94,115 @@ Now force push to update your PR:
 git push --force
 ```
 
+## API tests
+
+There are two kinds of tests in the API: unit (`phpunit`) and integration (`behat`) tests.
+
+Both `phpunit` and `behat` are development dependencies and should be available in the `vendor` directory.
```

Comment:
Add documentation links on the tools used, please.

`[behat](http://behat.org/en/latest/)` for example